### ISSUE TITLE
loss function on CRF layer

### ISSUE THREAD

**Issue by kakatrina** [tensorflow/addons#2313]
I want to apply a CRF layer on top on two-layered bi-lstm network for a NER model. But I figured the crf layer in the addon doesn't have a loss function implemented. I wonder how do I implement a loss function for training? Thanks ahead!

**Comment by WindQAQ** [comment]
Hi @kakatrina, can you take a look at https://github.com/tensorflow/addons/blob/master/tensorflow_addons/layers/tests/crf_test.py#L125-L164. Thank you!

**Comment by seanpmorgan** [comment]
Closing. Please let us know if the example in the test case is insufficient.
